l1interop/l1sseclient.go

```diff
@@ -224,8 +225,18 @@ func (l *l1SseClient) Start(nConnectedl1s *atomic.Uint64) error {
 			log.Errorw("error while reading l1 requests; will reconnect and retry", "err", err)
 		}
 
-		nConnectedl1s.Dec()
-		log.Infow("lost connection to L1", "l1", l.l1Addr, "nL1sConnected", nConnectedl1s.Load())
+		n = nConnectedl1s.Dec()
+		log.Infow("lost connection to L1", "l1", l.l1Addr, "nL1sConnected", n)
+		reportConnectionStatus(n)
+	}
+}
+
+func reportConnectionStatus(nConnectedl1s uint64) {
+	// Report the connection status to Filecoin Station
+	if nConnectedl1s > 0 {
+		fmt.Printf("INFO: Saturn Node is online and connected to %v peer(s)\n", nConnectedl1s)
+	} else {
+		fmt.Print("ERROR: Saturn Node lost connection to the network\n")
```

Comment:
```suggestion
		fmt.Print("ERROR: Saturn Node can't connect to the network\n")
```

Would this be more accurate in case the initial connect fails too?

Reply:
Good point! Let's defer that to #67 please.